src/ts/components/core/menu/Menu.tsx

```diff
@@ -46,7 +46,7 @@ const Menu = (props: Props) => {
             {React.Children.map(children, (child: any, index) => {
                 const childType = child.props._dashprivate_layout.type;
                 if (childType === "MenuTarget") {
-                    const { boxWrapperProps } = child.props;
+                    const { boxWrapperProps } = child.props._dashprivate_layout.props;
                     const boxProps = { w: "fit-content", ...boxWrapperProps };
```

Comment:
Not a big deal, but this could be simplified a bit - in the process reducing similar confusion in the future - by explicitly pulling out the real child props:
```js
const {type: childType, props: childProps} = child.props._dashprivate_layout;
if (childType === "MenuTarget") {
    const { boxWrapperProps } = childProps;
    return (
        <MantineMenu.Target key={index}>
            <Box w="fit-content" {...boxWrapperProps}>{child}</Box>
```

Reply:
Yes, that’s better - I’ll make the change before merging.
Thanks for the tip :-)